ambry-tools/src/main/java/com.github.ambry/store/DumpDataHelper.java

```diff
@@ -255,7 +255,7 @@ public BlobRecordInfo(String messageHeader, BlobId blobId, String blobProperty,
    * @throws IOException
    * @throws MessageFormatException
    */
-  private BlobRecordInfo readSingleRecordFromLog(RandomAccessFile randomAccessFile, long currentOffset)
+  public BlobRecordInfo readSingleRecordFromLog(RandomAccessFile randomAccessFile, long currentOffset)
```

Comment:
why is this `public` ? the class itself is package-private so this being `public` doesn't seem right.